test/test_bytes2human.py

```diff
@@ -81,7 +81,7 @@ def _create_testlist():
         values = [0, 1]
         for n in (1000, 1024):
             p = 1
-            for e in range(0, 6):
+            for _e in range(0, 6):
```

Comment:
Maybe better `_i` for consistency. I don't directly understand where the "e" is coming from, but "i" is typically a iterated counter.